packages/rlc-common/src/buildIndexFile.ts

```diff
@@ -299,9 +297,7 @@ function generateRLCIndex(file: SourceFile, model: RLCModel) {
     ]);
   }
 
-  // TODO: Disable LRO poller tentatively
-  // https://github.com/Azure/autorest.typescript/issues/2230
-  if (hasPollingOperations(model) && !model.options?.isModularLibrary) {
```

Comment:
will rlc code be different based on if it's modular or pure rlc?

Reply:
no